pkg/standalone/common.go

```diff
@@ -18,27 +18,32 @@ const (
 	defaultConfigFileName    = "config.yaml"
 )
 
-func defaultDaprDirPath() string {
+// DefaultDaprDirPath returns the default Dapr install path
+func DefaultDaprDirPath() string {
```

Comment:
once the dashboard code for standalone is moved to standalone pkg as suggested above, chnages to this file are not needed as they dont need to be exported. It will also reduce other changes related to these renames in the PR.